lib/puppet/provider/cs_order/crm.rb

```diff
@@ -93,9 +93,13 @@ def flush
     return if @property_hash.empty?
 
     updated = 'order '
-    updated << "#{@property_hash[:name]} #{@property_hash[:score]}: "
+    updated << "#{@property_hash[:name]} "
+    if @property_hash[:score]
+      updated << "#{@property_hash[:score]}: " unless @property_hash[:score].nil?
```

Comment:
if should already guard against `nil` so why is the `unless` necessary?

Reply:
Correct, will fix it.